src/Sylius/Bundle/CartBundle/spec/Doctrine/ORM/CartRepositorySpec.php

```diff
@@ -1,54 +0,0 @@
-<?php
-
-namespace spec\Sylius\Bundle\CartBundle\Doctrine\ORM;
-
-use Doctrine\ORM\AbstractQuery;
-use Doctrine\ORM\EntityManager;
-use Doctrine\ORM\Mapping\ClassMetadata;
-use Doctrine\ORM\Query\Expr;
-use Doctrine\ORM\QueryBuilder;
-use PhpSpec\ObjectBehavior;
-use Prophecy\Argument;
-use Sylius\Component\Cart\Model\CartInterface;
-use Sylius\Component\Order\Model\OrderInterface;
-
-class CartRepositorySpec extends ObjectBehavior
```

Comment:
It was easier, then refactoring this code. At the end, all repository logic is tested in scenarios